eZ/Bundle/EzPublishRestBundle/Tests/Functional/ContentTest.php

```diff
@@ -566,4 +567,25 @@ private function getVersionInfoFromJSONVersionListByStatus(array $versionList, $
 
         throw new \RuntimeException("Test internal error: Version with status {$status} not found");
     }
+
+    /**
+     * Assert that Version REST Response contains proper fields.
+     *
+     * @param \Buzz\Message\Response $response
+     */
+    private function assertVersionResponseContainsExpectedFields(Response $response)
+    {
+        $contentType = $response->getHeader('Content-Type');
+        self::assertNotEmpty($contentType);
+
+        $responseBody = $response->getContent();
+
+        // check if response is of an expected Content-Type
+        self::assertEquals('Version+xml', $this->getMediaFromTypeString($contentType));
+
+        // validate by custom XSD
+        $document = new \DOMDocument();
+        $document->loadXML($responseBody);
+        $document->schemaValidate(__DIR__ . '/xsd/Version.xsd');
```

Comment:
I expected to use here `/doc/specifications/rest/xsd/Version.xsd` however this does not validate. I'm not sure if I'm missing something or if this is some old spec. For now I've created my own XSD, which probably would need improvements when responses with more use cases are going to be validated.